_example/echo/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/arl/statsviz"
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	// Echo instance
+	e := echo.New()
+
+	// Statsviz
+	// Add monitor Go application runtime statistics (GC, MemStats, etc.)
+	// Create a new http ServeMux
+	mux := http.NewServeMux()
```

Comment:
I think the code speaks for itself in this case and does not need additional comments.
```suggestion
	mux := http.NewServeMux()
```